__tests__/books.test.js

```diff
@@ -0,0 +1,23 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('book routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('should return list of books', async () => {
```

Comment:
this test only gets a one book rather than a list — it can get confusing if your test description doesn't match what the test does